config/default.js

```diff
@@ -3,6 +3,7 @@ var apiUrl = 'http://localhost:' + apiPort; // eslint-disable-line no-var
 var frontendUrl = 'http://localhost:8080'; // eslint-disable-line no-var
 
 module.exports = {
+    appName: 'New App',
```

Comment:
Isn't it specific for the admin ? If it is, shouldn't we specify it in a specific admin entry ?
```
module.exports = {
    admin: {
        appName: 'New App',
```